SaintCoinach/ex.json

```diff
@@ -10113,7 +11493,7 @@
     },
     {
       "sheet": "TerritoryType",
-      "defaultColumn": "Name",
+      "defaultColumn": "PlaceName",
```

Comment:
Changed to displace a more readable result for example
"Limsa Lominsa Upper Decks" instead of "s1t1"

-Checked to ensure that it still works for every area